price_feed_poller.py

```diff
@@ -21,12 +21,17 @@ def handle_requestUpdate(w3, pricefeedcontract, wrbcontract, account_addr, gas,
     reward = wrbcontract.functions.estimateGasCost(w3.eth.gasPrice).call()
 
     # Hardcoded gas since it does not estimate well
-    dr_id = pricefeedcontract.functions.requestUpdate().transact(
-        {"from": account_addr, "gas": gas, "value": reward})
+    try:
+      dr_id = pricefeedcontract.functions.requestUpdate().transact(
+          {"from": account_addr, "gas": gas, "value": reward, "gasPrice": w3.eth.gasPrice}
```

Comment:
The gasPrice should be higher since with this value, at the moment it is been set to 1000000000, the transactions are failing with message` Result reward should cover gas expenses. Check the estimateGasCost method`